template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Nuget/.template.config/localize/templatestrings.cs.json

```diff
@@ -1,4 +1,4 @@
-﻿{
+{
```

Comment:
Could you please fix those whitespace changes? Thank you.

Reply:
Good catch. I don't know how these got changed :(